File: tests/java/sample.java
package app;

public class Account {
    static final double RATE = 0.05;
    private double balance;

    public Account(String id) {
        this.balance = 0;
    }

    public double balance() {
        return balance;
    }
}

interface Greeter {
    String greet();
}

enum Status { OPEN, CLOSED }

File: tests/csharp/sample.cs
namespace App;

public class Account
{
    public const double Rate = 0.05;
    private double _balance;

    public Account(string id) { }

    public double Balance()
    {
        return _balance;
    }
}

public interface IGreeter
{
    string Greet();
}

public enum Status { Open, Closed }
